library/systemd/src/lib/yast2/service_widget.rb

```diff
@@ -82,8 +82,8 @@ def content
           Left(
             HBox(
               Label(_("Current status:")),
-              Label(" "),
-              Label(Id(:service_widget_status), status)
+              HSpacing(1),
+              Label(Id(:service_widget_status), Opt(:hstretch), status)
```

Comment:
what is exactly problem? In past I see similar issue with CWM and correct solution should be usage of UI.RecalcLayout https://yast-ui-bindings.surge.sh/classYCP__UI.html#a7da317092ab6d90ca7bbfe06c196b54d

Reply:
You need to add a hook to the change event, moreover it recalculates the *whole* layout which is not trivial. This only affects the label.